apps/angular/src/app/features/anime/anime-management/anime-form/anime-form.component.ts

```diff
@@ -0,0 +1,161 @@
+
+	/** Description. */
+	readonly description: FormControl<string>;
+
+	/** Age rating. */
+	readonly ageRating: FormControl<AnimeRating>;
+
+	/** Source. */
+	readonly source: FormControl<AnimeSource>;
+
+	/** Season. */
+	readonly season: FormControl<AnimeSeason>;
+
+	/** Aired start date. */
+	readonly airedStart: FormControl<Date>;
+
+	/** Aired end date. */
+	readonly airedEnd: FormControl<Date>;
+
+	/** Image file. */
+	readonly imageFile: FormControl<ImageFile>;
+
+	/** Youtube trailer id. */
+	readonly youtubeTrailerId: FormControl<string | null>;
+
+	/** Genres. */
+	readonly genres: FormControl<MultipleAutocompleteItem[]>;
+
+	/** Genres. */
+	readonly studios: FormControl<MultipleAutocompleteItem[]>;
+}
+
+/** Anime form component. */
+@Component({
+	selector: 'camp-anime-form',
+	templateUrl: './anime-form.component.html',
+	styleUrls: ['./anime-form.component.css'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	providers: [
+		{
+			provide: MultipleAutocompleteService,
+			useClass: AnimeMultipleAutocompleteService,
+		},
+	],
+})
+export class AnimeFormComponent implements OnInit {
+
+	private readonly formBuilder = inject(NonNullableFormBuilder);
+
+	/** Anime types. */
+	protected readonly animeTypes = EnumUtils.toArray(AnimeType);
+
+	/** Anime statuses. */
+	protected readonly animeStatuses = EnumUtils.toArray(AnimeStatus);
+
+	/** Anime ratings. */
+	protected readonly animeRatings = EnumUtils.toArray(AnimeRating);
+
+	/** Anime sources. */
+	protected readonly animeSources = EnumUtils.toArray(AnimeSource);
+
+	/** Anime seasons. */
+	protected readonly animeSeasons = EnumUtils.toArray(AnimeSeason);
+
+	/** Anime form data group. */
+	protected readonly formGroup: FormGroup<AnimeManagementFormControls>;
+
+	/** Multiple autocomplete groups. */
+	protected get multipleAutocompleteGroups(): typeof animeMultipleAutocompleteGroups {
+		return animeMultipleAutocompleteGroups;
+	}
+
+	/** Anime form data. */
+	@Input()
+	public animeFormData: AnimeFormData | null = null;
+
+	/** Submit event. */
+	@Output()
+	public submitEvent = new EventEmitter<AnimeFormData>();
+
+	public constructor() {
+		this.formGroup = this.formBuilder.group({
+			englishTitle: ['', [Validators.required]],
+			japaneseTitle: ['', [Validators.required]],
+			type: [AnimeType.Unknown, [Validators.required]],
+			status: [AnimeStatus.NotYetAired, [Validators.required]],
+			isAiring: [false, [Validators.required]],
+			description: ['', [Validators.required]],
+			ageRating: [AnimeRating.Unknown, [Validators.required]],
+			source: [AnimeSource.Unknown, [Validators.required]],
+			season: [AnimeSeason.NonSeasonal, [Validators.required]],
+			airedStart: [new Date(), [Validators.required]],
+			airedEnd: [new Date(), [Validators.required]],
+			imageFile: ['' as ImageFile],
+			youtubeTrailerId: [null as string | null],
+			genres: [[] as MultipleAutocompleteItem[]],
+			studios: [[] as MultipleAutocompleteItem[]],
+		});
```

Comment:
```suggestion
		this.formGroup = this.formBuilder.group<AnimeManagementFormControls>({
			englishTitle: this.formBuilder.control('', Validators.required),
			...
		});
```

Use `formBuilder.control`. In addition, let's put the form initialization in a separate file.